organization/branding.md

```diff
@@ -0,0 +1,49 @@
+# The pyOpenSci brand
+
+The pyOpenSci brand is playful, curious, and integrates a calm color palette with illustrations and abstract primary shapes that are soft, round, and flowing.
+
+## Logos, colors, and fonts
+
+pyOpenSci stores its logos, brand colors, and font information in [Canva](https://www.pyopensci.org/handbook/organization/canva.html). We also store logos in a [publicly-shared Drive folder](https://drive.google.com/drive/folders/1m4cG3JLonP0mQ8y3pBaULMulpja-As0l?usp=drive_link).
+
+### pyOpenSci logos
+
+pyOpenSci uses three variations on a floral logo:
+* pyOpenSci written horizontally, with a flower "blooming" over the "O" in pyOpenSci
+* a four- or five-petaled flower with pyOpenSci abbreviated as pyOS
+* a five-petaled flower icon
+
+```{figure} /images/canva-images/pyos-flowers.png
+:alt: A series of four purple flower logos on grey backgrounds arranged in a single row. At the top of the collection is the text 'Logos', and underneath it is the text 'Assets(4).' From left to right, the logos include a dark purple logo with three petals above the word pyOpenSci. The 'O' in pyOpenSci is the center of the flower. The second image from the left is a four-petaled flower, with the text py and then an S in the shape of a snake inside an O. The text comes in from the left. The flower petals surround the O. The next image is identical to the second from the left, but is in a lighter shade of purple. The final image is the pyOpenSci flower, which has five petals and an open center. This logo is in light purple.
+
+A screenshot of the pyOpenSci flower logos in Canva.
+```
+
+### pyOpenSci color palette
+
+pyOpenSci's main color is purple, with the additional pink, green, and cream colors to be used as accents.
+
+```{figure} /images/canva-images/pyos-color-palette.png
+:alt: This is a series of color samples comprising the pyOpenSci color palette, which includes six equally-sized circles arranged in a single row. At the top of the image is the text 'Color palette.' The circles contain the following colors, with the corresponding hex code written beneath them: dark purple #33205C, medium purple #735FAB, light purple #BAB3D4, cream #F6F0DC, pink #BB92B0, and green #81C0AA.
+
+A screenshot of the pyOpenSci color palette in Canva.
+```
+
+### pyOpenSci fonts
+
+pyOpenSci uses [Itim](https://fonts.google.com/specimen/Itim) and [Poppins](https://fonts.google.com/specimen/Poppins?query=poppins) as its fonts. Itim is most commonly used as a header, while Poppins is used for the body of a text. Poppins is available in multiple weights, and at this time pyOpenSci uses the most appropriate weight for accessibility and readability, which varies by use case. Both of these fonts are available via [Google Fonts](https://fonts.google.com/).
+
+## Diversity, Equity, Inclusion, and Accessibility
```

Comment:
@kierisi Before I review this in its entirety, I'm curious—why is DEIA on a branding page? I could definitely see accessibility fitting in here as a statement that links to DEIA in our organizational values. I think you have some nice content here, but I don't think that the full DEIA statement belongs here. Rather, I'd put it in our org values and make a focused statement here around accessibility as it relates to font and alt tags. I haven't looked at the guidebook as a whole in a while, but I'm curious what your thoughts are on where this belongs in the handbook.

Reply:
@lwasser this is where it was listed in the google doc version of the handbook. happy to move it to organizational values if you'd prefer!
<img width="648" alt="Screenshot 2024-10-01 at 9 23 57 AM" src="https://github.com/user-attachments/assets/7d1ab883-653b-435b-b641-beccb29806c9">

Reply:
@kierisi i do understand that. That is an old document and things have been moved around a lot. It would be nice for you to consider where content belongs as you work on this as a part of this task. Please answer my question above.  

> I'm curious what your thoughts are on where this belongs in the handbook.

Do you think it makes sense in that section? If you don't, I'd greatly appreciate your considering the content relevance on this page and suggesting it go somewhere else as a part of the task of creating this handbook section.  I do think a note on accessibility makes sense here. Do you agree?

Reply:
it makes sense to me that DEIA language would go into branding, because it is a core component of our brand identity, which is why I didn't make suggestions for it to go elsewhere. 

I do think it makes sense to add more context on accessibility as well.

Reply:
That is reasonable. Please add more details about accessibility and how our brand addresses that

some examples: alt tags, responsive design, colors, fonts etc etc. Link out to other relevant sections as it makes sense - for example alt tags could link to the website section on adding figures that you worked on.